src/helpers/parser.ts

```diff
@@ -0,0 +1,53 @@
+import {
```

Comment:
Wraps our parser functions and makes it a bit easier to use. This essentially lazily loads the wasm bundle the first time you use it. It also has a nice function for validating table names and checking that you aren't trying to mix and match tables across testnets and mainnets.